README.md

```diff
@@ -1,135 +1,223 @@
-# Cronitor
+# Cronitor Ruby Library
 
 ![Test](https://github.com/cronitorio/cronitor-ruby/workflows/Test/badge.svg)
 [![Gem Version](https://badge.fury.io/rb/cronitor.svg)](https://badge.fury.io/rb/cronitor)
 
-[Cronitor](https://cronitor.io/) is a service for heartbeat-style monitoring of just about anything that can send an HTTP request.
 
-This gem provides a simple abstraction for the creation and pinging of a Cronitor monitor. For a better understanding of the API this gem talks to, please see [How Cronitor Works](https://cronitor.io/help/how-cronitor-works).
+[Cronitor](https://cronitor.io/) provides dead simple monitoring for cron jobs, daemons, data pipelines, queue workers, and anything else that can send or receive an HTTP request. The Cronitor Ruby library provides convenient access to the Cronitor API from applications written in Ruby.
 
-## Installation
+## Documentation
+See our [API docs](https://cronitor.io/docs/api) for a detailed reference information about the APIs this library uses for configuring monitors and sending telemetry pings.
 
-Add this line to your application's Gemfile:
+## Installation
 
-```ruby
-gem 'cronitor'
+```
+pip install cronitor
```

Comment:
Seeing this is a Ruby gem, I’m pretty sure `pip install` won’t work 😄

Reply:
lolol...i need to make another pass through the README! I started by copying over the python one :gri